src/d3d9/d3d9_device.h

```diff
@@ -1169,7 +1169,7 @@ namespace dxvk {
         const     uint32_t regCountHardware = DetermineHardwareRegCount<ProgramType, ConstantType>();
         constexpr uint32_t regCountSoftware = DetermineSoftwareRegCount<ProgramType, ConstantType>();
 
-        if (StartRegister + Count > regCountSoftware)
+        if (uint64_t(StartRegister) + Count > regCountSoftware)
```

Comment:
when are we ever going yo have more than 4 billion constants?

Reply:
When the application says we should write/read some there.

Reply:
why not just make this regCountSoftware - Count then

Reply:
That could underflow.